static/panes/compiler.ts

```diff
@@ -695,6 +707,13 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
             insertPoint.addChild(createOptView());
         });
 
+        this.stackUsageButton.on('click', () => {
```

Comment:
Can we make the stack usage button a drag source? Done above for the optButton as
```ts
        this.container.layoutManager
            .createDragSource(this.optButton, createOptView as any)
            // @ts-ignore
            ._dragListener.on('dragStart', togglePannerAdder);
```

Reply:
cool, didn't know about this feature.